### ISSUE TITLE
Release Twisted 15.1

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7758]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7758|
|Type|task|
|Created|2015-01-30 10:21:05Z|
|Branch|https://github.com/twisted/twisted/tree/releases/release-15.1.0-7758|

Keeping the pace up!

<details><summary>Searchable metadata</summary>

```
trac-id__7758 7758
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_releases_release_15_1_0_7758 branches/releases/release-15.1.0-7758
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1422613265086440 1422613265086440
changetime__1445217222500211 1445217222500211
version__None None
owner__hawkowl hawkowl
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

Builders spun, things done, review :)

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban commented|
|-|-|

Thanks for the branch!

Release notes looks good. +1 to merge it.

**Comment by twisted-trac** [comment]
|[<img alt="derwolfe's avatar" src="https://avatars.githubusercontent.com/u/641357?s=50" width="50" height="50">](https://github.com/derwolfe)<a name="note_6"></a>|@derwolfe set owner to @hawkowl|
|-|-|

The news files for the release look good to me as well.

Twistedchecker is returning 'new' errors that don't actually seem new. It seems to me that this is an issue with Twistedchecker and not with the new files related to the release. If you agree that the error lies with Twistedchecker, please go ahead and merge in the release.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl set status to `closed`|
|-|-|

(In [44384]) Merge releases/release-15.1.0-7758: Release Twisted 15.1

Author: hawkowl
Reviewers: adiroiban, herrwolfe
Fixes: [#7758](https://github.com/twisted/twisted/issues/7758)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl commented|
|-|-|

Ticket retargeted after milestone deleted
